autoPyTorch/pipeline/traditional_tabular_classification.py

```diff
@@ -147,7 +147,7 @@ def _get_hyperparameter_search_space(
                 to honor when creating the configuration space
             exclude (Optional[Dict[str, Any]]): what hyper-parameter configurations
                 to remove from the configuration space
-            dataset_properties (Optional[Dict[str, Union[str, int]]]): Characteristics
+            dataset_properties (Optional[Dict[str, Any]]): Characteristics
```

Comment:
Can this really be any? Should we define a DATASET_PROPERTIES_TYPE in the dataset file?

Then we can specify the valid types which I think are List/int/float/str?

Reply:
Yes this will be better.